### PR TITLE
FAQ - update 'various_device_problems' entry

### PR DESCRIPTION
**Pull request by heinezen**
Changelog:
- Put question into heading
- Rephrase german introduction to match english text

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -340,15 +340,14 @@
...
-                            "Einige Nutzer melden, dass sich ihr Smartphone manchmal seltsam verhält. Beispiel: Ab und zu bricht die Kommunikation zwischen dem Smartphone und Bluetooth-Geräten über andere Apps ab. Dies betrifft z. B. Smartwatches oder Bluetooth-Ohrhörer. Wir untersuchen diese Probleme, allerdings liegt es vermutlich an grundlegenden Funktionen des Betriebssystems, auf die wir keinen Einfluss haben.",
-                            "Versuchen Sie bitte Folgendes: <ul><li>Starten Sie Ihr Smartphone neu.</li><li>Entkoppeln Sie die anderen über Bluetooth verbundenen Geräte und koppeln Sie sie neu.</li></ul>",
+                            "Einige Nutzer melden, dass sich ihr Gerät seit der Installation der Corona-Warn-App manchmal seltsam verhält. Beispiel: Ab und zu bricht die Kommunikation zwischen dem Smartphone und Bluetooth-Geräten über andere Apps ab. Dies betrifft z. B. Smartwatches oder Bluetooth-Ohrhörer. Wir untersuchen diese Probleme, allerdings liegt es vermutlich an grundlegenden Funktionen des Betriebssystems, auf die wir keinen Einfluss haben.",
```
Comment (Ein-Tim) [inline src/data/faq_de.json:347]: I would not use "Ohrhörer" but rather "Kopfhörer".

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim @heinezen Thanks.

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
image

**Pull request merged**